README.md

```diff
@@ -153,3 +153,27 @@ try {
 ```
 
 See <https://stripe.com/docs/webhooks> and <https://stripe.com/docs/webhooks/signatures> for more information on setting up signed webhooks.
+
+
+### ColdBox Interception Points for Webhooks
+
+ColdBox users can automatically convert incoming webhooks to interception point events.  To do this,
+point Stripe at `/stripe-cfml/webhooks`.  `stripe-cfml` will take care of verifying the validity
+of incoming webhooks and announcing the associated interception point.
+
+Interception Points are a combination of `onStripe` and the camelCase version of the webhook type.
+For example, a `payment_intent.succeeded` type would become a `onStripePaymentIntentSucceeded` interception
+point.  A full list of interception points can be found in the `ModuleConfig.cfc` for this module.
+
+To use this feature, you will need to set your `endpointSecret` in your module settings:
+
+```cfc
+moduleSettings = {
+    "stripe-cfml": {
+        "apiKey": getSystemSetting( "STRIPE_API_KEY" ),
+        "endpointSecret": getSystemSetting( "STRIPE_ENDPOINT_SECRET" )
+    }
+};
+```
+
+More information about endpoint secrets can be found on [Stripe's website](https://stripe.com/docs/webhooks/signatures).
```

Comment:
This same link is referenced on line 155, so maybe this line is a little redundant?

Reply:
It is a bit redundant.  I tend to overdo it on links since people seem to only read certain sections of docs.  I can take it out if you'd like.

Reply:
Fair enough, that is true! You can leave it there 😀